README.md

```diff
@@ -171,6 +171,111 @@ See the [detailed query manager docs][query-manager-api] for more details on
 usage, or the [Apollo service API][apollo-service-api] if you need to use
 the service directly.
 
+### GraphQL Subscriptions
+
+GQL Subscriptions allow a client to subscribe to specific queries they are interested in tracking. The syntax for doing this is similar to `query` / `watchQuery`, but there are a few main differences:
+
+- ) you must define a `subscription` (versus a `query` or `mutation`)
+- ) because subscriptions are async by nature, you have to listen for these events and act accordingly.
+- ) subscriptions require websockets, so must configure your `link` accordingly
+
+**Creating your subscription:**
+
+`my-app/gql/subscription/new-human`
+
+```
+subscription {
+  newHuman() {
+    name
+  }
+}
+```
+
+**Subscribing from inside a route:**
+
+`app/routes/some-route.js`
+
+```js
+import Route from "@ember/routing/route";
+import { RouteQueryManager } from "ember-apollo-client";
+import query from "my-app/gql/subscription/new-human";
+
+export default Route.extend(RouteQueryManager, {
+  model() {
+    return this.get("apollo")
+               .subscribe({ query }, "human");
+  },
+
+  setupController(controller, model) {
+    model.on("event", event => alert(`${event.name} was just born!`));
+  },
+});
+```
+
+The big advantage of using the RouteQueryManager is that when you navigate away from this route, all subscriptions created will be terminated. That said, if you want to manually unsubscribe (or are not using the RouteQueryManager) `subscription.unsubscribe()` will do the trick.
+
+**Enabling Websockets**
+
+While this library should work w/ any back-end implementation, here's an example with Authenticated [Phoenix](https://github.com/phoenixframework/phoenix) + [Absinthe](https://github.com/absinthe-graphql/absinthe):
+
+`my-app/services/apollo.js`
+```js
+import ApolloService from 'ember-apollo-client/services/apollo';
+import { Socket } from 'phoenix';
+import { inject as service } from "@ember/service";
+import { createAbsintheSocketLink } from '@absinthe/socket-apollo-link';
+import AbsintheSocket from '@absinthe/socket';
+import { computed } from '@ember/object';
+
+
+export default ApolloService.extend({
+  session: service(),
+
+  link: computed(function () {
+    const socket = new Socket("ws://socket-url", {
+      params: { token: this.get('session.token') },
+    });
+    const absintheSocket = AbsintheSocket.create(socket);
+
+    return createAbsintheSocketLink(absintheSocket);
+  }),
+});
+
+```
```

Comment:
can you delete the newline at the end of the code sample and add one before the next paragraph?